src/schema/nodes/inline_equation.ts

```diff
@@ -18,27 +18,32 @@ import { ObjectTypes } from '@manuscripts/json-schema'
 import { NodeSpec } from 'prosemirror-model'
 
 import { ManuscriptNode } from '../types'
+import { CommentNode } from './comment'
 
 interface Attrs {
   id: string
```

Comment:
We don't need an id for inline math.

Reply:
What do you mean by "citations"?

Reply:
nevermind, my test was flawed, i removed the id